test/models/post_test.rb

```diff
@@ -26,7 +26,6 @@ class PostTest < ActiveSupport::TestCase
   should validate_presence_of(:body)
   should validate_presence_of(:kind)
   should validate_presence_of(:user)
-  should validate_length_of(:body).is_at_most(10_000)
```

Comment:
@johnthepink 

Yeah, removing it is the only option I suppose, the new `Should truncate body length if greater than 10,000` test covers this validation anyways.